### PR TITLE
Feature/replica bd

### PR DESCRIPTION
**Pull request by Gaviria9601**
update models to handle read.

### CODE REVIEW

File: dojo/routers/db_router.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
-from dojo.models import Product
+from dojo.models import Product, Engagement, Test
 
 class DbRouter:
     def db_for_read(self, model, **hints):
-        if model is Product:
+        if model in [Product, Engagement, Test]:
```
Comment (renejal) [inline dojo/routers/db_router.py]: If it is to validate the type? the isinstance method is used.

**Review by renejal — COMMENTED** [review]
changes

File: dojo/routers/db_router.py (hunk 1 above)
Comment (renejal) [inline dojo/routers/db_router.py]:
```suggestion
        if isinstance(model, (Product, Engagement, Test)):
```

**Review by renejal — COMMENTED** [review]
recommendation

File: dojo/routers/db_router.py (hunk 1 above)
Comment (Gaviria9601) [reply dojo/routers/db_router.py]: ready, updated

**Review by renejal — APPROVED** [review]

**Pull request merged**